src/types/api/users.ts

```diff
@@ -2,6 +2,9 @@ export interface UserReadSerializer {
   avatar: string | null;
   id: number;
   username: string;
+  default_wallet_balance?: number;
+  followers_count?: number;
+  following_count?: number;
 }
```

Comment:
Ideally, I think we should have a separate `UserStatsReadSerializer` but it was causing quite widespread impact, so I added these here instead and made them optional